packages/errors/src/codes.ts

```diff
@@ -1,3 +1,14 @@
+/**
+ * To add a new error, follow the instructions at
+ * https://github.com/solana-labs/solana-web3.js/tree/master/packages/error#adding-a-new-error
+ *
+ * WARNING:
+ *   - Don't remove error codes
+ *   - Don't change or reorder error codes.
+ */
+export const SOLANA_ERROR__TRANSACTION_MISSING_SIGNATURES = 69420 as const;
+export const SOLANA_ERROR__TRANSACTION_SIGNATURE_NOT_COMPUTABLE = 69421 as const;
```

Comment:
nit: I appreciate the joke and the fact that it's good not to start with 0 or 1 but I think it'd be best to start with a different "random" number because a user seeing an error like `Solana error #69420` bubbling up in production might not make the dApp look very professional.

Reply:
~For the culture!~

![Added via Giphy](https://media0.giphy.com/media/31Ix3PJm7fH9Va4szg/giphy.gif?cid=c9487743aeg1h6ufgyxxmqkfi51l29z77ehnt9m8cyiesp4p&ep=v1_gifs_search&rid=giphy.gif&ct=g)